gremlin-core/pom.xml

```diff
@@ -37,7 +41,7 @@ limitations under the License.
         <dependency>
             <groupId>org.yaml</groupId>
             <artifactId>snakeyaml</artifactId>
-            <version>1.15</version>
+            <version>${snakeyaml.version}</version>
```

Comment:
This version was initially used in the module gremlin-osgi-deps that I just removed (see https://github.com/gdelafosse/incubator-tinkerpop/commit/f29d01339514837ae4cc98240bc70f53b754aff7).
So indeed it's no more usefull here, but I usually centralize all the version in the dependencyManagment in the parent pom to see them all at a glance.
I'll put it back to its initial location as it's not your rule.